content/changes.md

```diff
@@ -8,20 +8,13 @@ summary: 'Learn about new features, fixes, and other improvements'
 
 Learn about notable new features, fixes, and other improvements to the application!
 
-## January, 2024 
+## January 31, 2024 
 
 ### Features 
  
 - **Learnosity Partnership Integration** 
```

Comment:
```suggestion
- **Qualified and Learnosity Integration** 
```

I'm flexible on this though, either way works. The link below says "partnership" so it seems a bit repetitive.